### ISSUE TITLE
VPAT 2.1.1a, Keyboard does not activate all items

### ISSUE THREAD

**Issue by david-iterators** [CityOfBoston/Iterators#15]
Screengrab/videos (insert file names here)
1, Keyboard, items missing when tabbing.mp4
2, Keyboard, Open menu and use cursor keys.mp4

Link
https://permit-finder.digital-staging.boston.gov

Environment
(Device > OS > Browser)
Computer > macOS 10.14.6 > Safari 12.1.2

Brief Description
Interactive elements are not accessible through the keyboard.

Steps to reproduce
1.  0:00 Look at “1, Keyboard, items missing when tabbing.mp4” 
Start pressing tab key to examine if all elements are accessible to keyboard users
2.  0:11  The Search element is not accessible

3.  0:00 Look at “2, Keyboard, Open menu and use cursor keys.mp4”, Start pressing the tab key to examine if all elements are accessible
4.  0:05 The menu item is not activated from the keyboard.  It is selected, but trying to press the enter or space keys do not activate the menu.
5.  0:10 The menu expands when doing shift-tab backwards to the menu.  Then the menu items are accessible.  This should be more accessible.

Desired Result
All items are accessible through the keyboard

Actual Result
Several items are not accessible.
[2, Keyboard, Open menu and use cursor keys.zip](https://github.com/CityOfBoston/Iterators/files/3467646/2.Keyboard.Open.menu.and.use.cursor.keys.zip)
[1, Keyboard, items missing when tabbing.zip](https://github.com/CityOfBoston/Iterators/files/3467647/1.Keyboard.items.missing.when.tabbing.zip)

**Comment by paylakatel** [comment]
This also seems to be a patterns/D8 issue with the menu/header - not permit finder, we're just pulling in this source code. I think it should be addressed when the entire patterns library is looked at.

**Comment by subaha-cob** [comment]
Wait to test on Production ONLY.

**Comment by subaha-cob** [comment]
Again wait for it to Patterns to move to production before testing.

**Comment by andrebtlima** [comment]
@subaha-cob can you remind me where we left this? I'm looking at the staging and production permit finder sites and looks like the search is still not accessible via keyboard. Also the tabbing order seems to be a little jumbled. After landing on the site and pressing tab, it should pull down the "Jump to Content" button but that doesn't seem to be highlighted until you've tabbed through nearly all the elements on the page.

**Comment by subaha-cob** [comment]
@andrebtlima The rest of this needs to be done within the App and not in Patterns. You will need to either reassign this to Phil or wait for when i am working on the the digital repo to fix it.

**Comment by subaha-cob** [comment]
@andrebtlima I have double checked this again. I have included all the changes on Boston.gov into patterns. Permit finder and the other sites should be working just like boston.gov. If they are not, the problem in several of them I checked was that they are not pulling in patterns library Javascript files. They need to pull or point to it for the menu to work correctly.

**Comment by subaha-cob** [comment]
@andrebtlima please test the bugs above on UAT. I pushed a fix for them today.

**Comment by subaha-cob** [comment]
@jill-iterators this is on the live site. So please test there unless I say otherwise. thanks.

**Comment by ollie-iterators** [comment]
This appears to be fixed
